spec/acceptance/suites/default/redis_multi_instances_one_host_spec.rb

```diff
@@ -34,29 +50,23 @@ class { 'redis':
     it { is_expected.to be_installed }
   end
 
-  describe service('redis-server-redis1') do
-    it { is_expected.to be_running }
-  end
-
-  describe service('redis-server-redis2') do
-    it { is_expected.to be_running }
-  end
-
-  describe file("#{config_path}/redis-server-redis1.conf") do
-    its(:content) { is_expected.to match %r{port 7777} }
-  end
+  instances.each do |instance|
+    describe file("/etc/systemd/system/redis-server-#{instance}.service"), if: (fact('service_provider') == 'systemd') do
+      its(:content) { is_expected.to match %r{redis-server-#{instance}.conf} }
+    end
 
-  describe file("#{config_path}/redis-server-redis2.conf") do
-    its(:content) { is_expected.to match %r{port 8888} }
-  end
+    describe service("redis-server-#{instance}") do
+      it { is_expected.to be_running }
```

Comment:
Mind adding `be_enabled` as well while you're at it?

Reply:
Sure, will do.